crates/sui-indexer/src/models/transactions.rs

```diff
@@ -80,7 +81,7 @@ impl TryFrom<SuiTransactionFullResponse> for Transaction {
     type Error = IndexerError;
 
     fn try_from(tx_resp: SuiTransactionFullResponse) -> Result<Self, Self::Error> {
-        let txn_json = serde_json::to_string(&tx_resp.transaction).map_err(|err| {
+        let tx_json = serde_json::to_string(&tx_resp.transaction).map_err(|err| {
```

Comment:
so `tx` going forward instead of `txn`?

Reply:
found out more tx in the codes, I was more used to txn though, so I will try to use tx to be consistent LOL

Reply:
ah i see, tbh i prefer `txn` too, maybe we need 2f+1 to determine :P